### ISSUE TITLE
Package.loadSettings is deprecated.

### ISSUE THREAD

**Issue by rtrigoso** [jroesch/language-scala#19]
Store package settings files in the `settings/` directory instead of `scoped-properties/`
```
Package.loadSettings (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:566:9)
<unknown> (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:168:43)
Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:147:15)
Package.load (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:161:12)
```

**Comment by optikfluffel** [comment]
* duplicate of #14 
* fixed in #12

**Comment by gangstead** [comment]
Fixed in `v1.0.1`
